sys-apps/firejail/firejail-0.9.62.4.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/netblue30/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="apparmor +chroot contrib debug +file-transfer +globalcfg +network +overlayfs +private-home +seccomp +suid test +userns vim-syntax +whitelist x11"
```

Comment:
Can `debug` be dropped? I can't find it used anywhere.

Reply:
Hm, sure looks like it. I'll do some verification, then update with `debug` removed.

Reply:
Confirmed I think; I can't find anything in the current release that cares about debug (no configure --enable-debug for instance). IUSE=debug was introduced in the firejail-0.9.60 ebuild but seems to serve no purpose; will strip it out.